wpiunits/src/main/java/edu/wpi/first/units/Current.java

```diff
@@ -4,6 +4,15 @@
 
 package edu.wpi.first.units;
 
+/**
+ * Unit of current dimension.
```

Comment:
Maybe specify this is for electric current? It'd read more clearly

```suggestion
 * Unit of electric current dimension.
```